### ISSUE TITLE
Difficulty reproducing training loss

### ISSUE THREAD

**Issue by plebbyd** [qinglew/PCN#32]
I'm using a modified version of the PCN model in order to train an AutoEncoder that essentially re-constructs the input:

- Input is the coarse ground truth pointcloud of the object
- Only use the Encoder + MLP+Re-shape portion of the PCN network
- Overall loss is just cd_loss_L1
- Using ModelNet10 dataset for training (bathtub, bed, chair, desk, dresser, monitor, night stand, sofa, table, and toilet)

The best loss (same value as CD) I am able to get is ~ 0.06. My question is: am I doing something wrong or missing a key component?

I would think that I should be able to achieve losses at least less than 1.Nx as this repository (10e-3). I am using a 3090 Ti with batch sizes anywhere from 256 to 1024.
